charts/ingress-nginx/templates/controller-service-internal.yaml

```diff
@@ -9,8 +9,8 @@ metadata:
   labels:
     {{- include "ingress-nginx.labels" . | nindent 4 }}
     app.kubernetes.io/component: controller
-  {{- if .Values.controller.service.labels }}
-    {{- toYaml .Values.controller.service.labels | nindent 4 }}
+  {{- if .Values.controller.service.internal.labels }}
+    {{- toYaml .Values.controller.service.internal.labels | nindent 4 }}
```

Comment:
This will break existing deployments. Please implement this in a backward compatible way.